plugins/com.google.cloud.tools.eclipse.appengine.validation/xslt/servlet.xsl

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	This stylesheet changes the servlet in web.xml to Servlet 2.5.
+-->
+<xsl:stylesheet version="1.0"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:ns1="http://xmlns.jcp.org/xml/ns/javaee"
+	xmlns:ns2="http://java.sun.com/xml/ns/javaee"
+	exclude-result-prefixes="ns1 ns2">
+
```

Comment:
Adding this causes my tests to fail: "com.sun.org.apache.xerces.internal.dom.DeferredTextImpl cannot be cast to org.w3c.dom.Element" at this line
`Element element = (Element) documentElement.getFirstChild();`

Looking into it